docs/assemblies/assembly-record-encryption-filter.adoc

```diff
@@ -0,0 +1,29 @@
+// file included in the following:
+//
+// index.adoc
+
+[id='assembly-record-encryption-filter-{context}']
+= Record Encryption filter
+
+[role="_abstract"]
+Streams for Apache Kafka Proxy's Record Encryption filter enhances the security of Kafka messages.
+The filter uses industry-standard cryptographic techniques to apply encryption to Kafka messages, ensuring the confidentiality of data stored in the Kafka Cluster.
+Streams for Apache Kafka Proxy centralizes topic-level encryption, ensuring streamlined encryption across Kafka clusters.
```

Comment:
`centralizes` or `centralises` ?

Reply:
Our DEV_GUIDE should call out whether the project is favo(u)ring the orthography of US or standard English.  We should then use it consistently across code and docs.

Reply:
We should use US English for docs and the website, because (I _think_, maybe @PaulRMellor can confirm?) that's the standard used in other Red Hat docs and to which we will be held.

Reply:
_theatrical resigned sigh_

fine.